### PR TITLE
(doc) Update build instructions

### PR DESCRIPTION
**Pull request by gep13**
## Description Of Changes

Add documentation on how to complete build of Chocolatey CLI.

## Motivation and Context

We got some feedback that is wasn't clear on how to build Chocolatey CLI

## Testing

New VM in Azure Portal was created, and steps executed as documented.

### Operating Systems Testing

N/A

## Change Types Made

* [ ] Bug fix (non-breaking change).
* [ ] Feature / Enhancement (non-breaking change).
* [ ] Breaking change (fix or feature that could cause existing functionality to change).
* [x] Documentation changes.
* [ ] PowerShell code changes.

## Change Checklist

* [x] Requires a change to the documentation.
* [x] Documentation has been updated.
* [ ] Tests to cover my changes, have been added.
* [ ] All new and existing tests passed?
* [ ] PowerShell code changes: PowerShell v2 compatibility checked?

## Related Issue

N/A

### CODE REVIEW

**Comment by gep13** [comment]
@flcdrg can I ask that you have a look at this PR, and provide any additional changes/suggestions?  Thanks!

File: setup.ps1
Comment (flcdrg) [inline setup.ps1:6]: The tests do need Pester 5.3.1 - should this be installed here? (And probably mention this in the README?

Comment (gep13) [reply setup.ps1:6]: Execution of the Pester tests is best done via the Vagrantfile that is provided, since it is possible that changes could be made to your system as a result of running the tests.  This is documented here: https://github.com/chocolatey/choco/blob/develop/TESTING.md#pester-tests

I notice that we don't have a link to this section from the README.md file, which we likely should have.

**Review by corbob — DISMISSED** [review]
These changes look good to me.

I'll hold off on merging in case there's any desire to tackle this comment in this PR:

> I notice that we don't have a link to this section from the README.md file, which we likely should have.

File: setup.ps1
Comment (gep13) [reply setup.ps1:6]: I have added a new section to the README.md to provide some context and links to the TESTING.md file.

**Comment by gep13** [comment]
@corbob as per discussions, I have updated this PR, and added a new section to link off to the TESTING.md file.

Can you have another review of this, and if you are happy, this can be merged in.

Thanks

**Review by corbob — APPROVED** [review]
LGTM

**Comment by corbob** [comment]
Thanks for getting this fixed up @gep13 👍

**Pull request merged**
